hi-ml-multimodal/test_multimodal/image/model/test_model.py

```diff
@@ -132,3 +136,23 @@ def test_reload_resnet_with_dilation() -> None:
     with torch.no_grad():
         expected_output = expected_model(image)
         assert torch.allclose(outputs_dilation, expected_output)
+
+
+@torch.no_grad()
+def test_hubconf() -> None:
+    """Test that instantiating the image model using the PyTorch Hub is consistent with older methods."""
+    image = torch.rand(1, 3, 480, 480)
+
+    github = 'microsoft/hi-ml'
+    model_hub = torch.hub.load(github, 'biovil_resnet', pretrained=True)
+    model_himl = get_biovil_resnet()
+
+    output_hub: ImageModelOutput = model_hub(image)
+    output_himl: ImageModelOutput = model_himl(image)
+
+    for field_hub, field_himl in zip(fields(output_hub), fields(output_himl)):
```

Comment:
Quick comment - couldn't you be missing some fields here when you zip them together? Why not just check the ones from the himl model?

Reply:
Addressed in a14767f.